libs/ui/tabs/brain/src/lib/brn-tabs-trigger.directive.ts

```diff
@@ -48,4 +48,12 @@ export class BrnTabsTriggerDirective {
 		if (!this._key) return;
 		this._root.setValue(this._key);
 	}
+
+	get isSelected(): boolean {
```

Comment:
maybe instead change it to a signal called _selected and expose a public readonly selected that is _selected.asReadonly()?

Reply:
@goetzrobin - I was just seeing computed signals are readonly by default. So could we not just do the following changes. See this PR https://github.com/thatsamsonkid/spartan/pull/2/commits/8b4676861ced76fc43e98e2931dcf09c3be7b539
Let me know what you think

Reply:
I like it! Much cleaner! Tabs was one of the earlier components so I am glad we are revisiting this and cleaning things up!

Reply:
Awesome merged those changes in